### PR TITLE
+dj/kill.m: update (add connection, order_by support) - fix #229

### LINKED ISSUE #229: output of dj.kill has no particular order
matlab equiv for datajoint/datajoint-python#668
(add order_by arg)

### PR DESCRIPTION
**Pull request by ixcat**
to address #229 
having issues with mym currently - draft until validation

### CODE REVIEW

**Comment by ixcat** [comment]
todo:
- support both single string and cell array of strings.

**Comment by ixcat** [comment]
Thanks @dimitri-yatsenko - I believe all comments are addressed in last commit.

**Comment by ixcat** [comment]
see also: https://github.com/datajoint/datajoint-matlab/issues/229

**Comment by ixcat** [comment]
^ should include docs-parts per https://github.com/datajoint/datajoint-docs/issues/237

File: docs-parts/computation/06-distributed-computing_kill_order_by.rst [hunk 1]
```diff
@@ -0,0 +1,14 @@
+
+For example, to sort the output by hostname in descending order:
+
+.. code-block:: text
```
Comment (guzman-raphael) [inline docs-parts/computation/06-distributed-computing_kill_order_by.rst:4]:
```suggestion
.. code-block:: matlab
```

File: docs-parts/computation/06-distributed-computing_kill_order_by.rst [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+
+
+    >> dj.kill('', dj.conn, 'host desc');
```
Comment (guzman-raphael) [inline docs-parts/computation/06-distributed-computing_kill_order_by.rst:7]:
```suggestion
    dj.kill('', dj.conn, 'host desc');
```

**Review by guzman-raphael — CHANGES_REQUESTED** [review]
Can you update the docs to use `matlab` highlighting?

File: docs-parts/computation/06-distributed-computing_kill_order_by.rst (hunk 2 above)
Comment (ixcat) [reply docs-parts/computation/06-distributed-computing_kill_order_by.rst]: adjusted in e5c655c

File: docs-parts/computation/06-distributed-computing_kill_order_by.rst (hunk 1 above)
Comment (ixcat) [reply docs-parts/computation/06-distributed-computing_kill_order_by.rst]: adjusted in e5c655c

**Pull request merged**